quickstarts/gradle/webapp-wildfly/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+    id 'war'
+    id 'org.eclipse.jkube.kubernetes' version '1.6.0-SNAPSHOT'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+
+group 'org.eclipse.jkube.kubernetes'
```

Comment:
All of our quickstarts seem to be following `org.eclipse.jkube.quickstart.gradle` group format. It would be nice if we can rename it in a similar fashion